.github/workflows/build.yml

```
@@ -341,8 +341,19 @@ jobs:
     strategy:
       matrix:
         profiles: ${{ fromJSON(needs.dynamic-matrix.outputs.profiles) }}
+        network:
+          - name: testnet
+            snapshot_url: https://snapshots.vara-network.io/vara-testnet.snap
+            snapshot_file: vara_ci_testnet.snap
+            disable_spec_name_check: true
+            run_checks_all: false
+          - name: mainnet
+            snapshot_url: https://snapshots.vara-network.io/vara-mainnet.snap
+            snapshot_file: vara_ci.snap
+            disable_spec_name_check: false
+            run_checks_all: true
```

```suggestion
            args: --checks all
```
No need to run `on-runtime-upgrade` twice